target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/grx390_dlink_dwr-966.dts

```diff
@@ -0,0 +1,288 @@
+/dts-v1/;
+
+#include "grx390.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "dlink,dwr-966", "lantiq,xway", "lantiq,grx390";
+	model = "D-Link DWR-966";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	aliases {
+		led-boot = &power_error_red;
+		led-failsafe = &power_error_red;
+		led-running = &power_green;
+		led-upgrade = &power_error_red;
+
+		led-internet = &web_green;
+		led-wifi = &wifi_2g_green;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 9 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+
+		ant {
+			label = "ant";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "green:wps";
+			gpios = <&stp 23 GPIO_ACTIVE_LOW>;
+		};
+		web_green: web-green {
```

Comment:
The node name should match the last part of the label, unless there are multiple leds which only deviate in color.

So, this would be just "internet", if there were multiple internet LEDs those would be "internet_green", "internet_red" etc.

The DT label (prefix in this line) will then have the same string prefixed by "led_", as this DT label is used on global context. So, all put together:

```
   led_internet: internet {
      label = "dwr-966:green:internet";
      gpios = <&stp 22 GPIO_ACTIVE_LOW>;
   };
```

However, since the DT label does not seem to be used anywhere, you could drop it and simply remove the "led_internet:" for this node.

Despite, please add an empty line between the led nodes.